bool framing
bool noise
bool rxOverrun
bool format
bool crc
